File: seed/csharp-sdk/exhaustive/oidc-token/src/SeedExhaustive/Endpoints/Params/Requests/CreateWithBodyAndQuery.cs
using global::System.Text.Json.Serialization;
using SeedExhaustive.Core;
using SeedExhaustive.Types;

namespace SeedExhaustive.Endpoints;

[Serializable]
public record CreateWithBodyAndQuery
{
    [JsonIgnore]
    public string? Fields { get; set; }

    [JsonIgnore]
    public required ObjectWithRequiredField Body { get; set; }

    /// <inheritdoc />
    public override string ToString()
    {
        return JsonUtils.Serialize(this);
    }
}

File: seed/ruby-sdk-v2/exhaustive/wire-tests/lib/seed/endpoints/params/types/create_with_body_and_query.rb
# frozen_string_literal: true

module Seed
  module Endpoints
    module Params
      module Types
        class CreateWithBodyAndQuery < Internal::Types::Model
          field :fields, -> { String }, optional: true, nullable: false, api_name: "_fields"

          field :body, -> { Seed::Types::Object_::Types::ObjectWithRequiredField }, optional: false, nullable: false
        end
      end
    end
  end
end
